yarn.lock

```diff
@@ -8273,7 +8273,7 @@ webidl-conversions@^3.0.0:
   resolved "https://registry.yarnpkg.com/webidl-conversions/-/webidl-conversions-3.0.1.tgz#24534275e2a7bc6be7bc86611cc16ae0a5654871"
   integrity sha1-JFNCdeKnvGvnvIZhHMFq4KVlSHE=
 
-webpack-cli@^4.6.0:
+webpack-cli@^4.9.0:
```

Comment:
Finally you didn't change any dependencies in `package.json` so this file should probably be reverted.